### DISCUSSION TITLE
请问有没有办法可以在局域网内访问alist

### DISCUSSION THREAD

**Discussion by lhflowersbloom** [AlistGo/alist#2156] [Q&A]
就是在浏览器内输入127.0.0.0:5244,直接登入alist，查看文件

**Comment by mixpplus** [comment]
在本机安装alist服务之后即可，访问地址 `127.0.0.1:5244`

**Reply by 12189108** [reply]
错误的，应确保路由器等未开启ip隔离的情况下，访问由路由器或交换机等赋予的内网ip地址及端口，而不是127.0.0.1。127开头的地址可以说是A类的保留地址，用作本地软件环回测试本主机的进程之间的通信之用。若主机发送一个目的地址为环回地址的IP数据报，则本机中的协议软件就处理数据报中的数据，而不会把数据报发送到任何网络。
